ElectricityBillApi.Tests/ElectricProviderTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using ElectricityBillApi.Models;
+using System;
+using Xunit;
+
+namespace ElectricityBillApi.Tests
+{
+    public class ElectricProviderTests
+    {
+        private ElectricProvider _sut;
+
+        public ElectricProviderTests()
+        {
+            _sut = new ElectricProvider();
+        }
+
+        [Fact]
+        public void CalculatePrice_CalculatesCorrectPrice()
+        {
+            // arrange
+            var loc1 = new Location()
```

Comment:
Is there any reason to have a location with just an x? Or does it logically always come with x, y and z? If the answer is yes, consider refactoring making the 3 coordinates readonly and be required by a contract (ctor)

Reply:
Good point! Added the ctor